aquadrone_sim_worlds/worlds/empty_underwater.world

```diff
@@ -32,9 +32,10 @@
     </light>
 
     <physics name="default_physics" default="true" type="ode">
-      <max_step_size>$(arg max_step_size)</max_step_size>
+      <!--TODO: Get these parameters to work-->
```

Comment:
These parameters had always caused an warning on the command line and were not properly processed. Previously the warning was ignored and the program would continue, but now it causes a crash. Commenting out for now